fvm/transaction.go

```diff
@@ -85,6 +85,14 @@ func (i *TransactionInvocator) Process(
 		return err
 	}
 
+	i.logger.Info().Msgf("(%d) ledger interaction were used by a transaction script (txhash: %s) ", st.InteractionUsed(), proc.ID.String())
```

Comment:
```suggestion
	i.logger.Info().Msgf("(%d) ledger interactions used by transaction (txID: %s) ", st.InteractionUsed(), proc.ID.String())
```

Maybe even use the logger fields to include the tx ID

Reply:
fixed at 7fe32a3